ethstorage/p2p/protocol/syncclient.go

```diff
@@ -187,9 +187,10 @@ type SyncClient struct {
 func NewSyncClient(log log.Logger, cfg *rollup.EsConfig, newStream newStreamFn, storageManager StorageManager, params *SyncerParams,
 	db ethdb.Database, m SyncClientMetrics, mux *event.Feed) *SyncClient {
 	ctx, cancel := context.WithCancel(context.Background())
-	maxFillEmptyTaskTreads = int32(runtime.NumCPU() - 2)
-	if maxFillEmptyTaskTreads < 1 {
-		maxFillEmptyTaskTreads = 1
+	if params.FillEmptyConcurrency > 0 {
+		maxFillEmptyTaskTreads = params.FillEmptyConcurrency
+	} else if runtime.NumCPU() > 2 {
```

Comment:
What if the numCPU < 2?

Reply:
its init value is 1; maxFillEmptyTaskTreads = 1